about.html

```diff
@@ -0,0 +1,112 @@
+                <li><a href="index.html" class="nav-item">About</a></li>
+                <li><a href="#" class="nav-item">Program</a></li>
+                <li><a href="#" class="nav-item">Join</a></li>
+                <li><a href="#" class="nav-item">Sponsor</a></li>
+                <li><a href="#" class="nav-item">News</a></li>
+                <li><a href="#" class="nav-item tech">Tech Campaign</a></li>
+            </ul>
+        </nav>
+    </header>
+
+    <section id="heros" class="heros">
+        <h1 class="hero-header-about">Technological Advancement <br> in AI </h1>
+        <p class="hero-text-about">
+            A conference that enlight the advancement of artificial intelligent in our Technology
+            with people over 120 counrties of the world, will be holding in March, In Nigeria.
+            We have lots of program setup for this event and we will announcing new parntership
+        </p>
+
+        <p class="text-small-about">
+            Please contact us via email for any further questions
+            <span>ehmatech@gmail.com </span>  
+        </p> 
+    </section>
+
+    <section id="about-logo">
+        <h2 class="about-header">Technological Advancement <br> Logo</h2>
+        <div class="bar"></div>
+        <p class="about-text">
+            The logo of this company was decided through a logo competition from 3 April 2017. After online public voting the selection 
+            committee's pick a winner, and Emmanuel Essien won the prize. After much consultation with the board, we decide that this year's logo should be this:
+        </p>
+        <strong class="about-logo about-log">
+            <i class="fa fa-snowflake-o" aria-hidden="true"></i><span>Ehma</span>Tech
+        </strong>
+
+        <hr>
+    </section>
+        
+    <section id="past-submit">
+        <h2 class="past-submit-header">
+            See past AI conferences
+        </h2>
+        <div class="bar"></div>
+        <p class="submit-text">
+            Take a look at our past AI conferences that took place in Cario and in Istanbul
+        </p>
+        <div class="past-submit-imgs">
+            <div id="past-img">2019 <br> Technological Advancement <br> in Cario</div>
+            <div id="past-imgs">2019 <br> Technological Advancement <br> in Istanbul</div>
+        </div>
+       
+
+    </section>
+
+    <section id="partners">
+        <h2> Partners</h2>
+        <div class="bar"></div>
+
+        <div class="partners-logo">
+            <img src="imgs/partner1.webp" alt="Google" class="google" width="150" height="80">
+            <img src="imgs/partner2.png" alt="Toptal" class="toptal" width="150" height="80">
+        </div>
+    </section>
+
+    <footer id="footer">
+        <strong class="about-logo logo footer-logo">
+            <i class="fa fa-snowflake-o flake" aria-hidden="true"></i><span>Ehma</span>Tech
+        </strong>
+
+        <p class="footer-text"> 2017 Lorem ipsum dolor sit, amet dolor sit,amet 
```

Comment:
- [ ] Kindly make sure to remove the Partners section for `about desktop version`.
- [ ] Make sure that the footer section has a black background for `about section` as per the attached loom video.

Reply:
I have implemented these changes.